### DISCUSSION TITLE
Info Leak

### DISCUSSION THREAD

**Discussion by LimitlessPotential** [vincode-io/Zavala#170] [Support] [answered]
According to the AppStore page no information is collected from the app however I’ve just seen an advert on insta for one of the items from my shopping list.

Suspicious

**Comment by vincode-io** [comment, marked as answer]
Zavala is an Open Source app that can be audited for data gathering. It can also have its network traffic snooped for communication with third-party services. You should do some basic research like this before making baseless allegations.

Zavala does not collect or sell any user data. Period.
